<launch>
    <rosparam command="load" file="$(find lidar_detect)/cfg/qt_params.yaml"/>
    <node name="qt_detect_node" pkg="lidar_detect" type="qt_detect_node" />
    
    <include file="$(find obb_generator)/launch/obb_generator_launch.launch" />
</launch>